Central-Server/API/sanitizers.py

```diff
@@ -0,0 +1,11 @@
+import re
+# Just a bunch of helper functions to help with sanitizing and verification
+
+def is_hex(s):
+    return re.fullmatch(r"^[0-9a-fA-F]+", s or "") is not None
+
+def is_github_username(s):
+    return re.fullmatch(r"^[a-zA-Z0-9\-\_]+$", s or "") is not None
+
+def is_alphanum(s):
+    return re.fullmatch(r"^[a-zA-Z0-9\-\_]+$", s or "") is not None

```

Comment:
is there a point to having a separate regex for this when ur `is_github_username` handles this and python has a built in `.isalnum()`?

Reply:
@zyunlam

Reply:
I will rename this to is_github_username. This isn't exacly isalnum because we have a few other characters that we need to allow for.

Reply:
sounds good. another thing I should mention is that the regex should also include a slash since ideally branches look like `AV-12345/abc` or something to that effect. ig the article u sent mentions slashes too